<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionDialog</class>
 <widget class="QDialog" name="CollectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>292</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/screens/form_find.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="rateLbl">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>218</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>178</y>
     <width>182</width>
     <height>108</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources/images/images.qrc">
     <normaloff>:/images/screens/button_close_f.png</normaloff>:/images/screens/button_close_f.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>182</width>
     <height>108</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="leftBtn">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>58</y>
     <width>43</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources/images/images.qrc">
     <normaloff>:/images/screens/button_left.png</normaloff>:/images/screens/button_left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="rightBtn">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>58</y>
     <width>43</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resources/images/images.qrc">
     <normaloff>:/images/screens/button_right.png</normaloff>:/images/screens/button_right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="cardLbl3">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>35</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/cards/p.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cardLbl5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>35</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/cards/p.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cardLbl1">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>35</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/cards/p.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cardLbl2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>35</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/cards/p.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cardLbl4">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>35</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources/images/images.qrc">:/images/cards/p.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="indexLbl1">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>109</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indexLbl2">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>109</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indexLbl3">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>109</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indexLbl4">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>109</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="indexLbl5">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>109</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>released()</signal>
   <receiver>CollectionDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
